Fix state related race-condition on form save effects

NickPhura opened the pull request:
Some general improvements/fixes to the way the page loads and what happens when save actions occur.

NickPhura committed:
Fix state related race-condition on form save effects

sdevalapurkar commented on app/src/features/home/search/SearchActivityPage.tsx:
```diff
@@ -125,7 +127,7 @@ const SearchActivityPage: React.FC<ISearchActivityPage> = (props) => {
    */
   const onFormChange = useCallback(
-    debounced(500, (event: any) => {
+    debounced(100, (event: any) => {
```
question:
why is this value lowered?

NickPhura replied:
I arbitrarily chose 500 to begin with, but found it was too long. 

Specifically, there was a funny interaction where if you changed the form, and then within 500 milliseconds updated the map, it would blip back to the old version of the form for a moment, because the map change would trigger a re-render before the 500 milliseconds delay expired. Likely wouldn't occur that much in real life, but lowering this number should fix that.

It is certainly something we can adjust as we play with it more.

sdevalapurkar reviewed (dismissed):
lgtm

NickPhura committed:
Fix postman tests
- Mainly just had to re-add the postman user to the database and grant it the admin role
